src/views/requestAccess.js

```diff
@@ -1,144 +1,123 @@
 
-export const RoleDropDown = () => {
-	const [isActive, setIsActive] = useState(true);
-	const [dropContent, setDropContent] = useState("");
-	const [viewPropertyManagerOptions, setPropertyManagerOptions] = useState(false);
-	const [viewJoinStaffOptions, setJoinStaffOptions] = useState(false);
-
-	const handleClickDropdown = () => {
-		if (!isActive) setIsActive(true);
-	}
-
-	const handleClickJOIN = () => {
-		// Close dropdown
-		if (isActive) setIsActive(false);
-		// Set dropdown button text
-		setDropContent("JOIN Staff");
-		// Use JOIN staff checkbox
-		if (viewPropertyManagerOptions) setPropertyManagerOptions(false);
-		if (!viewJoinStaffOptions) setJoinStaffOptions(true);
-	}
-
-	const handleClickProperty = () => {
-		// Close dropdown
-		if (isActive) setIsActive(false);
-		// Set dropdown button text
-		setDropContent("Property Manager");
-		// Use property search box
-		if (viewJoinStaffOptions) setJoinStaffOptions(false);
-		if (!viewPropertyManagerOptions) setPropertyManagerOptions(true);
-	}
-
-	const bottomPad = (viewPropertyManagerOptions) ? "bottom-padding-property" : (viewJoinStaffOptions ? "bottom-padding-join" : "bottom-padding");
-	const dropDown = "dropdown is-rounded " + ((isActive) ? "is-active" : "");
-	const dropButton = "icon is-small " + ((viewPropertyManagerOptions) ? "drop-button-prop" : (viewJoinStaffOptions ? "drop-button-join" : "drop-button-start"));
-
-	return (
-		<div className={bottomPad} >
-			<div className={dropDown}>
-			    <div className="dropdown-trigger">
-				    <button className="button is-rounded drop-button" aria-haspopup="true" aria-controls="dropdown-menu" onClick={handleClickDropdown} >
-					    <span className="drop-button-content"> {dropContent} </span>
-					    <span className={dropButton} >
-					    	<i className="fas fa-angle-down" aria-hidden="true"></i>
-					    </span>
-				    </button>
-			    </div>
-			    <div className="dropdown-menu dropdown-menu-place">
-				  	<div className="dropdown-content dropdown-content-width">
-					    <DropDownItem dropContent="JOIN Staff" clickHandle={handleClickJOIN} />
-					    <DropDownItem dropContent="Property Manager" clickHandle={handleClickProperty} />
-				  	</div>
-			    </div>
-			</div>
-			<div>
-				{viewPropertyManagerOptions && <PropertyManagerOptions />}
-				{viewJoinStaffOptions && <JoinStaffOptions />}
-			</div>
-		</div>
-	);
+const RoleDropDown = (props) => {
+  console.log(props.selectionOptions);
+  console.log(typeof props.selectionOptions);
+  let roles = props.selectionOptions;
+  return (
+    <div className="section-row">
+      <div className="select is-rounded">
+        <select
+          defaultValue='default'
+          onChange={e => props.selectionHandler(e.target.value)}>
+          <option value='default' disabled defaultValue>Select Role</option>
+          {
+            props.selectionOptions.map((role) => 
+            <option>{role}</option>
```

Comment:
One other thing: When we do a .map or generate JSX content like this we need to add a "key" attribute to the `<option>` tag like `<option key={index}>` (you can access index from the map function as a parameter: .map( (role, index) => ...)

You can see the error it produces in the console if we don't do this